apm-opentracing/src/main/java/co/elastic/apm/opentracing/ApmScopeManager.java

```diff
@@ -54,18 +55,46 @@ private void doActivate(@Nullable Object span, Object traceContext) {
     }
 
     @Override
+    public Scope activate(Span span) {
+        return activate(span, false);
+    }
+
+    @Deprecated
     @Nullable
     public ApmScope active() {
+        final ApmSpan apmSpan = activeApmSpan();
+        final ApmScope apmScope;
+        if (apmSpan != null) {
+            apmScope = new ApmScope(false, apmSpan);
+        } else {
+            final Object traceContext = getCurrentTraceContext();
```

Comment:
Isn't that redundant and already handled in the `else` block of `activeApmSpan`?

Reply:
Yes, seems so...